### ISSUE TITLE
Output: Ensure constant 5 columns. Improve over variable 4 or 5 columns

### ISSUE THREAD

**Issue by freedom-isnotanarchy** [anchore/grype#617]
OUTPUT: Ensure constant 5 columns. Improve over variable 4 or 5 columns
WHY: Hard to parse and interpret, the current variable output.
REQUESTED:
```
NAME              INSTALLED     FIXED-IN      VULNERABILITY        SEVERITY
jcc               11.5.5.0      _             CVE-2007-5090        High
log4j-api         2.13.3        2.17.0        GHSA-p6xc-xr62-6r2g  High
log4j-api         2.13.3        _             CVE-2021-44832       Medium
log4j-core        2.13.3        2.15.0        GHSA-jfh8-c2jp-5v3q  Critical
log4j-core        2.13.3        _             CVE-2021-44228       Critical
log4j-core        2.13.3        2.16.0        GHSA-7rjr-3q55-vv33  Critical
```
TODAY:
```
NAME              INSTALLED     FIXED-IN      VULNERABILITY        SEVERITY
jcc               11.5.5.0                    CVE-2007-5090        High
log4j-api         2.13.3        2.17.0        GHSA-p6xc-xr62-6r2g  High
log4j-api         2.13.3                      CVE-2021-44832       Medium
log4j-core        2.13.3        2.15.0        GHSA-jfh8-c2jp-5v3q  Critical
log4j-core        2.13.3                      CVE-2021-44228       Critical
log4j-core        2.13.3        2.16.0        GHSA-7rjr-3q55-vv33  Critical
```

**Comment by freedom-isnotanarchy** [comment]
I suggest that, outputting a variable number of fields, is some small barrier to Grype adoption by users.

For example, even neophytes can get "some mileage", just using Bash to parse, today. Like this:
```
FIXES_AVAILABLE=$(        cat "${OUTPUT_FROM_SCAN}" | awk '{print "FIX:",$3,"VUL:",$4,"SEV:",$5}'    | cut -f6 -d " "  | sed '/^$/d' | wc -l ) )
FIXES_AVAILABLE_UNIQ=$(   cat "${OUTPUT_FROM_SCAN}" | tr -s " " | awk '{print $3,"#",$4,"#",$5,"#"}' | grep -v ' #  #' | sort -u     | wc -l ) )
```

But the more you try to do with, what Should Be, a very simple and dependable .CSV type of delimited format ?
That drives you to consider using Ruby, Python, etc. ... when all may be trying to do, is simple work.

In Summary: Today, we have an "Almost Table".  
If only, it was a reliable and "Actual Table" ... then some new audiences might open up,  to using and evangelizing Grype.

Thank you for your consideration.

**Comment by kzantow** [comment]
You could just use templating to output an alternate format -- see the docs: https://github.com/anchore/grype#using-templates ; does this help?

**Comment by freedom-isnotanarchy** [comment]
Thank you kzantow.  Based on your guidance, I've created a PR to simply address this "enhancement", with this documentation change, below.

This pull request: https://github.com/anchore/grype/pull/619

Alters only 1 file: https://github.com/freedom-isnotanarchy/grype/blob/doc-fix-enhancement-617-by-improving-templated-output-example/README.md

To address this enhancement: https://github.com/anchore/grype/issues/617

**Comment by luhring** [comment]
My two cents: I agree w/ @kzantow, the table output is intended to be a highly simplified view for humans, not for programs. The JSON output can be used in shell scripts with [`jq`](https://stedolan.github.io/jq/), and the template output is probably plenty flexible.
